Makefile

```diff
@@ -103,8 +100,14 @@ GO_TEST_FORMAT = pkgname
 
 ifeq ($(CI), true)
 	VERBOSE = true
+	OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell hack/get_ocp_versions_for_testing.sh))
+else
+	OPENSHIFT_VERSIONS := $(or ${OPENSHIFT_VERSIONS}, $(shell hack/get_ocp_versions_for_testing.sh | jq 'with_entries(select(.value.default))'))
 endif
 
+# RHCOS_VERSION should be consistent with BaseObjectName in pkg/s3wrapper/client.go
+RHCOS_BASE_ISO := $(shell (jq -n '$(OPENSHIFT_VERSIONS)' | jq '[.[].rhcos_image]|max'))
```

Comment:
Why is there a single "RHCOS_BASE_ISO" when there are multiple versions?

Reply:
(unrelated to PR)

Reply:
Podman onprem solution worked only with a single live ISO.
But it seems to be an irrelevant remainder since https://github.com/openshift/assisted-service/pull/1177 .
I'll remove it on a following PR.